### DISCUSSION TITLE
Is generative AI the right path to the end of AGI

### DISCUSSION THREAD

**Discussion by zhoupenger** [community/community#143553] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is generative AI the right path to the end of AGI, what do you guys think?

**Comment by JL-ghcoder** [comment]
However, AGI is not just about generating data; it requires a deeper understanding and the ability to generalize knowledge across different domains, contexts, and tasks in a way similar to human cognition. Its really hard to answer right now.

**Comment by pr0f3ss** [comment]
Human cognition means being able to infer logical consequences from an axiomatic ruleset. This process of deductive reasoning allows us to form proofs. Hence, I would broadly define cognizance as the ability to verify and validate your conjectures. 

Most LLMs currently only predict the statistically most probable token in line, based on your input and the general context. However, there has been some work to improve the logical problem solving skills of LLMs, such as in [Logic-LM](https://aclanthology.org/2023.findings-emnlp.248/). Another interesting paper in this context can be found [here](https://arxiv.org/abs/2306.09841).
